### ISSUE TITLE
`limiter` attribute lost in pickling

### ISSUE THREAD

**Issue by ValueRaider** [JWCook/requests-ratelimiter#99]
I'm experimenting with passing a rate-limiter object to processes, and encountered a pickling bug:

```
from requests_ratelimiter import LimiterSession
from pyrate_limiter import Duration, RequestRate, Limiter
history_rate = RequestRate(1, Duration.SECOND)
limiter = Limiter(history_rate)
session = LimiterSession(limiter=limiter)

import pickle
pickled_session = pickle.dumps(session)
unpickled_session = pickle.loads(pickled_session)
unpickled_session.limiter

```

**Comment by JWCook** [comment]
I believe this is because [requests.Session](https://github.com/psf/requests/blob/0e322af87745eff34caffe4df68456ebc20d9068/src/requests/sessions.py#L810-L816) overrides the [pickle state methods](https://docs.python.org/3/library/pickle.html#object.__getstate__) to only save the [specific attributes defined here](https://github.com/psf/requests/blob/0e322af87745eff34caffe4df68456ebc20d9068/src/requests/sessions.py#L375-L388). I don't know the reason for that, but I am guessing it's meant to exclude some connection state info that either can't be serialized or isn't useful.

Solutions for this would be either:
* Ignore `requests.Session`'s attribute whitelist, and add a `LimiterSession.__getstate__()` and __setstate__()` that uses the default `object` implementation. Not sure if that would have any consequences, but it would be worth testing.
* Set a `LimiterSession.__attrs__` class attribute that extends `requests.Session.__attrs__` with the additional attributes [listed here](https://github.com/JWCook/requests-ratelimiter/blob/65a1055afee15642a0a91d51198838508838f471/requests_ratelimiter/requests_ratelimiter.py#L67-L77)

Is that something you'd like to submit a PR for? If not, I can make those changes within the next couple weeks.

**Comment by ValueRaider** [comment]
I attempt a PR.

**Comment by JWCook** [comment]
Your fixes are now in v0.7.0 on PyPI. Thank you!
